internal/util/bundleutil/bundleutil.go

```diff
@@ -196,16 +197,35 @@ func (meta *BundleMetaData) BuildBundleImage(tag string) error {
 
 	img := fmt.Sprintf("%s:%s", meta.BaseImage, tag)
 
+	// switch back to current working directory, so that subsequent
+	// bundle version images can be built.
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if err := os.Chdir(cwd); err != nil {
+			log.Error(cwd)
+		}
+	}()
+
+	if err := os.Chdir(filepath.Dir(meta.BundleDir)); err != nil {
+		return err
+	}
+
 	if len(meta.BuildCommand) != 0 {
 		// TODO(varsha): Make this more user friendly by accepting a template which
 		// can executed in each bundle subdirectory.
-		log.Infof("Using the specified command to build image %s", img)
-		cmd := exec.Command(meta.BuildCommand, img)
+		log.Infof("Using the specified command to build image")
+		commandArg := strings.Split(meta.BuildCommand, " ")
+
+		cmd := exec.Command(commandArg[0], commandArg[1:]...)
```

Comment:
This still needs `img` as an arg

Reply:
thinking about this, after the tag the build context is always going to be the current directory. So the input becomes, `docker build -f bundle.Dockerfile` in the `build-cmd` tag. The user can anyway not change the path of dockerfile, so why not include in the `commandArg`?